guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -88,16 +104,19 @@ To create a pull request do the following:
 
 1.  Fill out the form, and click **Create pull request** again to submit the pull request&mdash;that’s it!
 
-## Report an issue
-If you find a typo or erroneous information in Magento DevDocs, you can either fix it with a pull request (as described above) or you can report it by creating an issue in the DevDocs Github repository.
+## Report an issue {#issues}
+If you find a typo or erroneous information in Magento DevDocs, you can either fix it with a pull request (as described above) or you can report it by creating an issue in the DevDocs Github repository. New GitHub issues include questions and requests for information to help us write or update content. Fill in as much information as you can.
```

Comment:
"GitHub" not "Github"

Reply:
Maybe "Submitted GitHub issues should include questions and requests for information that assist us in writing or updating content."

Reply:
revising - meant the new issue will use a template requesting all that info.